sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorLogExporter.cs

```diff
@@ -44,19 +44,26 @@ public override ExportResult Export(in Batch<LogRecord> batch)
             // Prevent Azure Monitor's HTTP operations from being instrumented.
             using var scope = SuppressInstrumentationScope.Begin();
 
+            ExportResult exportResult = ExportResult.Failure;
+
             try
             {
                 var telemetryItems = LogsHelper.OtelToAzureMonitorLogs(batch, LogResource, _instrumentationKey);
-                var exportResult = _transmitter.TrackAsync(telemetryItems, false, CancellationToken.None).EnsureCompleted();
+                if (telemetryItems.Count > 0)
```

Comment:
Do we need a similar change in the other Exporters?

Reply:
Yes, was planning in separate one but just pushed the changes here.